pom.xml

```diff
@@ -220,6 +220,15 @@
                 </exclusion>
             </exclusions>
         </dependency>
+        <dependency>
```

Comment:
I don't think this is needed as a new dependency, private-api-sdk-java is already in the pom

<private-api-sdk-java.version>2.0.174</private-api-sdk-java.version>